app/views/layouts/application.html.erb

```diff
@@ -11,10 +11,17 @@
   </head>
 
   <body>
+
+    <%= render 'global/topbar' %>
+    
+    <%= render 'global/header', locals: { title: pageTitle } %>
```

Comment:
The fix for having the title working is: `render partial: 'global/header' ...` :)
I'll add it to the PR and merge it